services/API-service/src/api/event/trigger-per-lead-time.entity.ts

```diff
@@ -42,8 +42,8 @@ export class TriggerPerLeadTime {
   public eventName: string;
 
   @Column({ default: false })
-  public triggered: boolean;
+  public forecastAlert: boolean;
 
   @Column({ default: true })
-  public thresholdReached: boolean;
+  public forecastTrigger: boolean;
```

Comment:
Let's add a note to help our future selves to interpret this variable.

```
public forecastTrigger: boolean; // true indicates a trigger for this lead time, false indicates either a warning or no event REFACTOR: to either use this for all disaster types or to remove this exception for floods
```

This looks like it's user-facing via the api so possibly update the wiki to help others also interpret this variable.

Reply:
commend included
this particular entity property itself is not yet user-facing right? There's also no existing wiki documentation on this atm. I will add a link to this PR comment in the documentation task of this item.